frontend/webpack.config.ts

```diff
@@ -373,9 +373,10 @@ const config: Configuration = {
 
 if (CHECK_CYCLES === 'true') {
   new CircularDependencyPreset({
-    exclude: /node_modules|public\/dist/,
+    exclude: /node_modules|public\/dist|\.(gql|html)$/,
+    // TODO: investigate how to load the plugins registry asynchronously
+    filterModules: /node_modules\/@console\/active-plugins\.js/,
```

Comment:
Could this not be part of the `exclude` regex instead of needing a separate option? How is the behavior different between `exclude` and `filterModules`?

Reply:
`excludeModules` does not exclude any cycles where a module is involved but is not the primary file that the plugin is worried about. e.g., this module is not excluded by `exclude`, even though `node_modules/@console/active-plugins.js` is one of the links in the cycle (and thus would've met the `exclude` regex)

Good point about it being ambiguous though, add some TSdoc to clear it up

```
console/frontend/packages/console-app/src/plugin.tsx
  packages/console-app/src/plugin.tsx
  public/module/k8s/index.ts
  public/module/k8s/get-resources.ts
  public/plugins.ts
  node_modules/@console/active-plugins.js
  packages/console-app/src/plugin.tsx
```